governance.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run by volunteers in a collaborative and open way. Portions of this governance policy have been inspired by that of the Apache Software Foundation.
+
+# The R-Multiverse Project
+
+The R-Multiverse project consists of the code and repositories that reside in the `r-multiverse` GitHub organization and the r-multiverse.org website. All repositories are open to everyone, with the exception of a private repository where security and conduct-related matters are shared between the administrators only.
+
+## R-Multiverse Volunteers
+
+We recognize four groups of individuals related to the project.
+
+### User
+
+A user is someone who uses R-Multiverse by downloading and installing packages from the R-Multiverse repositories. They may also contribute to R-Multiverse by providing feedback in the form of bug reports and feature suggestions in the `r-multiverse/help` repository. Users participate in the R-multiverse community by helping other users in `r-multiverse/help` or in other external forums.
```

Comment:
I feel the user/contributor words are not very adapted since both categories can contribute. How about users vs developers? or package users vs package authors?

Reply:
I'll work on this section to delineate a bit more. The idea is that users are just anyone who may use R-Multiverse in `install.packages()`, whereas a contributor is someone who actively interacts with the R-Multiverse infrastructure by submitting a package. The obligations / expectations on the former are then much lower.